test/jsonapi/plugs/query_parser_test.exs

```diff
@@ -140,8 +140,8 @@ defmodule JSONAPI.QueryParserTest do
   end
 
   test "get_view_for_type/2 raises on invalid fields" do
-    assert_raise InvalidQuery, "invalid fields, comment for type mytype", fn ->
-      get_view_for_type(MyView, "comment")
+    assert_raise InvalidQuery, "invalid fields, comments for type mytype", fn ->
```

Comment:
Per the above deprecation pathway, this test should be noted as the deprecated pathway and an additional test added to show the sanctioned way forward.

Reply:
Added a third test to capture the three cases and label on the deprecated test case.